untitled/src/task6.java

```diff
@@ -0,0 +1,27 @@
+import java.util.Scanner;
+
+//Tasks
+public class task6 {
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        System.out.println("Please enter 1 four-digit natural number from the keyboard that falls within the range 1000 – 9999");
+        int number = scan.nextInt();
+
+        int a = number / 100;
```

Comment:
You need to slip the numbe rinto 4 digits and to create a new 2 numbers from them:
1st digit & 4th digit
2nd digit & 3rd digit

Input:
8435 from this create 85 and 43

Output:
85 > 43